gammapy/irf/irf_map.py

```diff
@@ -0,0 +1,120 @@
+from astropy.io import fits
+from gammapy.maps import Map
+import numpy as np
+from copy import deepcopy
+
+
+class IRFMap:
+    """IRF map base class"""
+    def __init__(self, irf_map, exposure_map):
+        self._irf_map = irf_map
+        self.exposure_map = exposure_map
+
+    @classmethod
+    def from_hdulist(cls, hdulist):
```

Comment:
As discussed in the last developer call, I added the option back to specify the hdu name on read.

In general I think we should be very strict on writing and not allow to specify different HDU names to avoid that we get incompatibilities. And some point we should probably write up a convention and document it...